### PR TITLE
Certificate rotation and custom certificates

### PR DESCRIPTION
**Pull request by galal-hussein**
https://github.com/rancher/docs/issues/1111

### CODE REVIEW

File: content/rke/v0.1.x/en/certificate-management/_index.md [hunk 1]
```diff
@@ -0,0 +1,174 @@
+---
+title: Certificate Management
+weight: 150
+aliases:
+  - /rke/v0.1.x/en/installation/certificate-management/
+---
+
+## Certificate Rotation
+
+As of v0.2.0, RKE can be used to rotate the cluster certificates with different options, the certificate rotation is one of the subcommands of `./rke cert` command:
```
Comment (alena1108) [inline content/rke/v0.1.x/en/certificate-management/_index.md]: I feel like we need to explain on how to rotate certs from Rancher UI too.

**Comment by deniseschannon** [comment]
can you rebase off of `staging` @galal-hussein

File: content/rke/v0.1.x/en/certificate-management/_index.md (hunk 1 above)
Comment (galal-hussein) [reply content/rke/v0.1.x/en/certificate-management/_index.md]: done

**Comment by alena1108** [comment]
LGTM

**Pull request merged**